src/index.js

```diff
@@ -49,6 +50,10 @@ export function target(target) {
 }
 
 export function transform(source, options = {}) {
+	// We clone the options object because we mutate it later on. If we don't, then
+	// issues crop up if the caller re-uses the same options object between calls
+	// to this function.
+	options = clone(options);
```

Comment:
`options = { ...options }` and you don't need an extra dependency

Reply:
should be equivalent to a shallow clone

Reply:
They don't use object spread anywhere in this repo, I didn't want to introduce it in case it was intentional (for support of older node versions perhaps)